lms/static/scripts/frontend_apps/components/dashboard/CourseActivity.tsx

```diff
@@ -32,6 +32,7 @@ type AssignmentsTableRow = {
  */
 export default function CourseActivity() {
   const { courseId } = useParams<{ courseId: string }>();
+  const { organizationPublicId } = useParams();
```

Comment:
You can merge these two calls to `useParams()`:

```suggestion
  const { courseId, organizationPublicId } = useParams<{ courseId: string; organizationPublicId?: string }>();
```